config/manifests/bases/fence-agents-remediation.clusterserviceversion.yaml

```diff
@@ -7,8 +7,10 @@ metadata:
     categories: OpenShift Optional
     containerImage: ""
     createdAt: ""
-    description: Fence Agents Remediation Operator for remediating nodes using upstream
-      fence-agents.
+    description: Fence Agents Remediation Operator uses well-known agents to fence
+      and remediate unhealthy nodes. In this process it can minimize downtime for
+      stateful applications, restores compute capacity if transient failures occur,
+      and increases the availability of workloads.
```

Comment:
Nit: "availability" sounds like uncountable to me, so I'd say "improve", more than "increase"